tests/installation/accept_license.pm

```diff
@@ -28,11 +28,11 @@ use warnings;
 use base 'y2_installbase';
 use y2_logs_helper qw(verify_license_has_to_be_accepted verify_license_translations accept_license);
 use testapi;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_caasp);
 
 sub run {
     my ($self) = @_;
-    assert_screen('license-agreement', 120);
+    assert_screen('license-agreement', 120) unless is_caasp('suse_microos');
```

Comment:
but it suppose to have some license agreement , right ? in this case I would post a bug and record_softfailure here

Reply:
It has, this just skips this needle for MicroOS cause the screen where it shows the license is different. Later in the code where license is accepted is executed.

Reply:
> > Before some questions rise, here some answers:
> > 
> > 1. I am extending the function `is_caasp` instead of creating a new one (e.g. `is_suse_microos`). This function is used in openSUSE MicroOS too and the functionality is completely valid for SUSE MicroOS too. Maybe it's a bit misleading, but if we create a new function it should cover both enterprise and openSUSE microOS's and leave `is_caasp` as a CaaSP only product.
> 
> and I prefer it more . I mean having `is_microos` function . Because it was confusing before your PR and with it is even more confusing at least from my POV

If others agree too, I'll create `is_microos` only for SUSE MicroOS for now and keep openSUSE microos where it is, otherwise I would touch a lot of jobs in O3. Later on, openSUSE MicroOS check can be moved to this new function too.

Reply:
> > > Before some questions rise, here some answers:
> > > 
> > > 1. I am extending the function `is_caasp` instead of creating a new one (e.g. `is_suse_microos`). This function is used in openSUSE MicroOS too and the functionality is completely valid for SUSE MicroOS too. Maybe it's a bit misleading, but if we create a new function it should cover both enterprise and openSUSE microOS's and leave `is_caasp` as a CaaSP only product.
> > 
> > 
> > and I prefer it more . I mean having `is_microos` function . Because it was confusing before your PR and with it is even more confusing at least from my POV
> 
> If others agree too, I'll create `is_microos` only for SUSE MicroOS for now and keep openSUSE microos where it is, otherwise I would touch a lot of jobs in O3. Later on, openSUSE MicroOS check can be moved to this new function too.

We can actually transform  `is_caasp` to  `is_microos` as CaaSP is not tested in openQA any more AFAIK.

Reply:
> We can actually transform is_caasp to is_microos as CaaSP is not tested in openQA any more AFAIK. But that might be another future task.

For some reason, there are some tests that use `is_caasp('caasp')` in the tests section (e.g. [this](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/microos/one_line_checks.pm#L79) and [this](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/transactional/transactional_update.pm#L105))

Maybe @Vogtinator can give more insight here.

Reply:
AFAICT those are leftovers, from back when CaaSP used openQA and can be removed.